src/components/CreateVmWizard/CreateVmWizard.js

```diff
@@ -62,6 +62,22 @@ function getInitialState (clusters, templates, blankTemplateId) {
       templateId = blankTemplateId
     }
   }
+  const blankTemplateMap = templates.get(blankTemplateId)
+  if (blankTemplateMap !== undefined) {
+    const memory = blankTemplateMap.get('memory')
+    const osName = (blankTemplateMap.get('os')).get('type')
+    const operatingSystemMap = operatingSystems.find(os => { return localeCompare(os.get('name'), osName) === 0 })
+    const operatingSystemId = operatingSystemMap.get('id')
+    const cpus = (blankTemplateMap.get('cpu')).get('vCPUs')
+    const initEnabled = blankTemplateMap.get('cloudInit').get('enabled')
+    DEFAULT_STATE.steps.basic = {
+      ...DEFAULT_STATE.steps.basic,
+      operatingSystemId: operatingSystemId,
+      memory: memory / (1024 ** 2), // MiB
+      cpus: cpus,
+      initEnabled: initEnabled,
+    }
+  }
```

Comment:
Some style suggestions and using ImmutableJS slightly different:

```suggestion
  const blankTemplate = templates.get(blankTemplateId)
  let blankTemplateValues = {}
  if (blankTemplate) {
    const osName = (blankTemplate.get('os')).get('type')
    const operatingSystem = operatingSystems.find(os => os.get('name') === osName) 
    const operatingSystemId = operatingSystem.get('id')
    const memoryInB = blankTemplate.get('memory')
    const cpus = blankTemplate.getIn([ 'cpu', 'vCPUs' ])
    const initEnabled = blankTemplate.getIn([ 'cloudInit', 'enabled' ])
    
    blankTemplateValues = {
      operatingSystemId,
      memory: memoryInB / (1024 ** 2), // bytes to MiB
      cpus,
      initEnabled,
    }
  }
```

then below `blankTemplateValues` can be merged in on it's own

Reply:
```jsx
  const state = merge(
    {},
    DEFAULT_STATE.
    { steps: { basic: blankTemplateValues } },
    {
      ...
    })
```